### ISSUE TITLE
Test Livewire component with inside a bladewind component (No such file or directory)

### ISSUE THREAD

**Issue by babeuloula** [mkocansey/bladewind#147]
Hello,

I try to test a Livewire component with the select component inside and I've got this error: 

```
include_once(vendor/bladewind/js/select.js): Failed to open stream: No such file or directory (View: /srv/vendor/mkocansey/bladewind/resources/views/components/select.blade.php) (View: /srv/vendor/mkocansey/bladewind/resources/views/components/select.blade.php)

  at storage/framework/views/01820bd273d27dcf646d34658b57c1bd9b06d397.php:417
    413▕     <h1><?php echo __DIR__ ?></h1>
    414▕ </div>
    415▕ 
    416▕ <script>
  ➜ 417▕ <?php include_once('vendor/bladewind/js/select.js'); ?>
    418▕ const bw_<?php echo e($input_name); ?> = new BladewindSelect('<?php echo e($input_name); ?>', '<?php echo e($placeholder); ?>');
    419▕ <?php if(!$disabled && !$readonly): ?> bw_<?php echo e($input_name); ?>.activate(); <?php endif; ?>
    420▕ </script>
    421▕ <?php /**PATH /srv/vendor/mkocansey/bladewind/src/../resources/views/components/select.blade.php ENDPATH**/ ?>
```

<details>
<summary>My code is realy easy</summary>

```blade
<div>
    @if(\count($list) > 0)
        <x-bladewind::card :title="My users"
           reduce_padding="true"
           has_shadow="true"
           class="shadow-sm my-4"
        >
            <x-bladewind::select
                id="user-selector"
                required="true"
                searchable="true"
                name="user_id_select"
                selected_value="{{ $selected }}"
                label_key="name"
                value_key="id"
                image_key="picture"
                data="{{ json_encode($list) }}"
            />
        </x-bladewind::card>
    @endif
</div>
```

```php
Livewire::test('users.user-selector')
```
</details>

The script file is load from the view folder instead of the public folder.

Thanks,
Regards

**Comment by mkocansey** [comment]
Hello @babeuloula 

The error suggests the `select.js` file is not in your public directory.

What version of BladewindUI are you on? 
If not the latest, you will need to update to the latest (2.2.8) to get the select component.

If you are already on the latest version, kindly run the command below from the root of your project and let me know if it works.

`php artisan vendor:publish --provider="Mkocansey\Bladewind\BladewindServiceProvider" --tag=bladewind-public --force`

**Comment by mkocansey** [comment]
Hello @babeuloula 

The select component is wired to load its script file from the `public` directory to confirm with Laravel standards.

In your case if you prefer to load the scripts from the view folder you will need to edit the `select.blade.php` and change `line 163` 
```php
@php include_once('vendor/bladewind/js/select.js'); @endphp
```

You need to change the path to where you have the script file.

**Comment by babeuloula** [comment]
I've already execute the command `php artisan vendor:publish --provider="Mkocansey\Bladewind\BladewindServiceProvider" --tag=bladewind-public --force`. I don't want to load from the view folder, it's just a suggestion because when I edit the cached view, the select.js try to load from the view root directory and not the public directory.

**Comment by babeuloula** [comment]
No idea how to resolve. And you can replicate with a Livewire test ?

**Comment by mkocansey** [comment]
Sure. I will take a look into this.

**Comment by haugen86** [comment]
Sorry for opening a year old issue @mkocansey  😅

But I have an old project I've installed Bladewind UI on, and after installation a lot of my tests started failing. More accurately the ones that's actually visiting pages in my application. I got the same kind of error as @babeuloula, and it comes from the tests trying to find the javascript files in the vendor folder from root, and not the `public`folder.

`<script>
    @php include_once(public_path('vendor/bladewind/js/notification.js')); @endphp
</script>`

Adding the `public_path` helper solved it for me. I can do a PR which would add this helper to wherever js is loaded in the view components.
